runtime/oti/JITInterface.hpp

```diff
@@ -123,6 +123,15 @@ typedef struct {
 	U_64 fpr[32];
 } J9JITRegisters;
 
+#elif defined(J9VM_ARCH_RISCV)
+
+typedef struct {
+	union {
+		UDATA numbered[32];
```

Comment:
Please add a member of this union (like x86) that allows reference by name rather than just comments like on lines 250 and 270 below (i.e. so we can say `gpr.named.RA` rather than `gpr.numbered[1]`).

Reply:
Updated the code as suggested above.